packages/types/lib/index.ts

```diff
@@ -405,6 +406,11 @@ export interface Rect {
   height: number;
 }
 
+export interface NewWindow {
+  handle: string;
+  type: string;
```

Comment:
could `type` be anything, or is it only a handful of different strings?  if the latter, list them in a union:

```ts
  type: 'foo' | 'bar' | 'baz'
```